### PR TITLE
Update NATS server to 2.10.4

### PR DESCRIPTION
**Pull request by bruth**

### CODE REVIEW

**Review by wallyqs — APPROVED** [review]
LGTM

File: tests/run-images.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -ex
 
-ver=(NATS_SERVER 2.9.23)
```
Comment (wallyqs) [inline tests/run-images.sh:4]: I think this one has to be `2.9.23` actually

File: tests/run-images-2019.ps1 [hunk 2]
```diff
@@ -3,7 +3,7 @@ Set-PSDebug -Trace 2
 # Exit on error.
 $ErrorActionPreference = "Stop"
 
-$ver = "NATS_SERVER 2.9.23".Split(" ")[1]
```
Comment (wallyqs) [inline tests/run-images-2019.ps1:6]: leave as `2.9.23`?

File: tests/run-images.sh (hunk 1 above)
Comment (bruth) [reply tests/run-images.sh:4]: Hrm. Let me check, I think the tests run on which ever version is currently being released. At least that is how it worked before.

File: tests/build-images-2019.ps1 [hunk 3]
```diff
@@ -3,7 +3,7 @@ Set-PSDebug -Trace 2
 # Exit on error.
 $ErrorActionPreference = "Stop"
 
-$ver = 'NATS_SERVER 2.9.23'.Split(' ')[1]
```
Comment (wallyqs) [inline tests/build-images-2019.ps1:6]: also has to remain `2.9.23` I think

File: tests/run-images.sh (hunk 1 above)
Comment (bruth) [reply tests/run-images.sh:4]: There are only one set of tests for both versions so which is being released is the one that would replace this value. The 2.9.x one has not changed, so no need for testing that.

I believe you did a dual release before correct, so both versions needed to be tested?

Comment (bruth) [reply tests/run-images.sh:4]: For posterity, noting @wallyqs and I spoke and no need to change since only one version bump is happening.

**Pull request merged**